src/App.jsx

```diff
@@ -1,10 +1,18 @@
+import Login from "./components/Authentication/Login/Login";
+import Signup from "./components/Authentication/Signup/Signup";
 import "./tailwind.css";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
 
 function App() {
   return (
-    <div className="bg-gray-800 text-center w-full h-screen">
-      <h1>Interview Project Dev</h1>
-    </div>
+    <BrowserRouter>
+      <div className="bg-white text-center w-full h-screen">
```

Comment:
if we want this to be applied over all pages we can create a separate file called Layout.jsx 

[This is how it should look here : ](https://reactrouter.com/en/main/start/concepts#layout-routes)
And the way the Layout component is structured is simple: 

```js 
export const Layout = () => {

return <div className="bla bla classes">
   // any other component you want to be rendered in the layout like for eg: 
   <Sidebar />
   // and the outlet which you can import [from here](https://reactrouter.com/en/main/components/outlet)
   <Outlet />
<div>
}```